save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/TestController.kt

```diff
@@ -33,6 +33,18 @@ class TestController {
         executionId?.let { testExecutionService.saveTestExecution(executionId, testsIds) }
     }
 
+    /**
+     * @param executionId ID of the [Execution], during which these tests will be executed
+     * @param testSuiteId ID of the [TestSuite], for which there will be created execution in DB
+     */
+    @Suppress("UnsafeCallOnNullableType")
+    @PostMapping("/getAllTestsByTestSuiteIdAndSaveExecution")
+    fun getAllTestsByTestSuiteIdAndSaveExecution(@RequestBody executionId: Long, @RequestParam testSuiteId: Long) {
```

Comment:
Also, why is endpoint called `get...`, if it's `POST` and returns nothing?

Reply:
Because it also get tests from DB by `testSuiteId`, however it could be confusing, renamed